yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -176,6 +199,32 @@ func packAndSendAssessMigrationPayload(status string, errMsg error) {
 		obfuscatedIssues = append(obfuscatedIssues, obfuscatedIssue)
 	}
 
+	/*
+		Case to skip for sql statement anonymization:
+			1. if issue type is unsupported query construct or unsupported plpgsql object
+			2. if object type is view or materialized view
+			3. if sql statement is empty
+	*/
+	for i, issue := range assessmentReport.Issues {
+		// TODO: add a hidden flag to enable anonymization for Unsupported Query Constructs
+		skipCondition1 := slices.Contains([]string{UNSUPPORTED_QUERY_CONSTRUCTS_CATEGORY, UNSUPPORTED_PLPGSQL_OBJECTS_CATEGORY, UNSUPPORTED_DATATYPES_CATEGORY},
+			issue.Category)
+		skipCondition2 := slices.Contains([]string{constants.VIEW, constants.MATERIALIZED_VIEW, constants.TRIGGER, constants.FUNCTION, constants.PROCEDURE}, issue.ObjectType)
+		skipCondition3 := issue.SqlStatement == ""
+
+		if skipCondition1 || skipCondition2 || skipCondition3 {
+			continue
+		}
+
+		// NOTE: indexing/ordering needs to be same in obfuscatedIssues and assessmentReport.Issues
```

Comment:
This is a little error-prone and confusing. 
1. We first iterate through assessmentReport.Issues, to create obfuscatedIssues. 
2. Then iterate through assessmentReport.Issues again, but modify obfuscatedIssues[i].SqlStatement with the assumption that both the lists are 1:1? 
3. Finally, pass obfuscatedIssues to the callhome struct


Can we just do all of it in one single loop? 
- create obfuscated issue
- anonymize Sql statement
- append to obfuscatedIssues